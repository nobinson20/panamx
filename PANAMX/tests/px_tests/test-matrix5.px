int main() {
    matrix m;
    matrix n;
    m = [1.6, 3.9;
         2.4, 4.3;
         5.7, 7.2];
    n = [12.3, 15.6, 14.5, 18.8;
         23.4, 21.8, 9.5,  10.7];
    m = m * n;
    printm(m);
    free(m);
    free(n);
    return 0;
}
