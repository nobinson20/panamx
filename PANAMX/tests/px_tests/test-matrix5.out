[
	110.940   109.980   60.250   71.810   
	130.140   131.180   75.650   91.130   
	238.590   245.880   151.050   184.200   
]
